nss-day-cohort-45/tabloidfullstack-giving-gillyflowers
# User Profile List and Details

### Changes:
#### Back-End:
- added methods to `UserProfileRepository` and `UserProfileController` for getting all user profiles and getting them by user profile id
#### Front-End:
- added methods to `UserProfileProvider` to get all users and get user by id
- added `UserProfileList`, `UserProfileCard`, and `UserProfileDetails` components
- added routes for above to `Application Views`
---
### Testing:
- fetch branch and run app and API
- log in and select User Profile Management from the Admin Menu
- you should see a list of all user profiles, sorted alphabetically by Display Name
- clicking on a user's display name should take you to the User Profile Details page, which should show their avatar (if present), their display name, their full name, their email, their creation date, and their user type
